### PR TITLE
add Crispy HUD

### PR DESCRIPTION
**Pull request by rfomin**
woof0000

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
> Should the face be centred or removed?

I don't care. I never liked it, but people seem to prefer it to be visible. I'd say it looks even worse on the gun sprite.

File: src/mn_menu.c [hunk 1]
```diff
@@ -1798,7 +1798,7 @@ static void M_SizeDisplay(int choice)
         default:
             break;
     }
-    screenblocks = BETWEEN(3, 12, screenblocks);
+    screenblocks = BETWEEN(3, 13, screenblocks);
```
Comment (fabiangreffrath) [inline src/mn_menu.c:1801]: What if a PWAD contains an SBARDEF lump that does not provide the extra HUD?

**Comment by rfomin** [comment]
> > Should the face be centred or removed?
> 
> I don't care. I never liked it, but people seem to prefer it to be visible. I'd say it looks even worse on the gun sprite.

Current state:
woof0001

File: src/mn_menu.c (hunk 1 above)
Comment (rfomin) [reply src/mn_menu.c:1801]: Should be OK. We will show the last available status bar.

File: src/mn_setup.c [hunk 2]
```diff
@@ -1818,7 +1818,7 @@ static void RefreshSolidBackground(void)
 static const char *screensize_strings[] = {
     "",           "",           "",           "Status Bar", "Status Bar",
     "Status Bar", "Status Bar", "Status Bar", "Status Bar", "Status Bar",
-    "Status Bar", "Fullscreen", "Fullscreen"
+    "Status Bar", "Fullscreen", "Crispy",     "None"
```
Comment (rfomin) [inline src/mn_setup.c:1821]: This is problematic because SBARDEF entries don't have names.

**Comment by fabiangreffrath** [comment]
> Current state:

Well, now it's full frontal on the weapon sprite.

File: src/mn_setup.c (hunk 2 above)
Comment (rfomin) [reply src/mn_setup.c]: Fixed in cf17054

**Comment by rfomin** [comment]
Should we merge this?

**Review by fabiangreffrath — APPROVED** [review]
Please merge, thank you!

**Pull request merged**